docs/intro.md

```diff
@@ -80,9 +85,22 @@ The above code is perfectly valid, with a minimal resolver meeting the requireme
 
 Real-world resolvers are rarely that simple: they usually read data from some source such as a database, process inputs, or resolve value in the context of a parent object. How should our basic resolver look to resolve a client's user agent?
 
-In Ariadne every field resolver is called with at least two arguments: `obj` parent object, and the query's execution `info` that usually contains the `context` attribute that is GraphQL's way of passing additional information from the application to its query resolvers.
+In Ariadne every field resolver is called with at least two arguments: the
+query's parent object, and the query's execution `info` that usually contains
+a `context` attribute. The `context` is GraphQL's way of passing additional
+information from the application to its query resolvers.
+
+> In the above example, note the `*_` argument in the resolver's method signature.
+> The underscore is a convention used in many languages (including Python) to
+> indicate a variable that will not be used. The asterisk prefix is Python syntax
+> that informs the method it should expect a variable-length argument list. In
+> effect, the above example is throwing away any arguments passed to the resolver.
+> We've used that here, to simplify the example so that you can focus on its
+> purpose.
```

Comment:
Please remove explicit linebreaks from this quote block.

We could also bolden first sentence in this block using strong emphasis syntax (`**text**`) and then put blank line between it and rest of the block - this produces fancy "titled yellow box" effect without introducing headers into hierarchy.